### ISSUE TITLE
Command line bugging on startup

### ISSUE THREAD

**Issue by smarbal** [packing-box/docker-packing-box#15]
When launching the Packing Box, the command line is bugging visually (e.g. character inputs are not shown.).
The issue can be solved after pressing `Ctrl + C` which will yield this output : 
```
^C
[1]+  Done                    sudo Xvfb "${DISPLAY}" -screen 0 "${SCREENSIZE}x16"
```

**Comment by dhondta** [comment]
Hi @smarbal !
Good catch. I did not manage to find why it now messed up with STDIN when starting up the box but it seems it comes from the Xframe Virtual Buffer (Xvfb, the virtual X server for managing GUI-based applications from the terminal) when started with `sudo `.

**Comment by dhondta** [comment]
@smarbal 
OK, I got it. This issue indeed comes from the fact the `Xvfb` command is run with `sudo`, hence allowing to create a temporary folder with owner `root`. This is aimed to, from what I could find, get file handles open for listening.
While running the command without `sudo` then, we got error "*_XSERVTransmkdir: Owner of /tmp/.X11-unix should be set to root*" that shows this fact.
From [here](https://github.com/dimkir/nightmare-lambda-tutorial/issues/10), I could find a fix by adding some options to not listen and therefore not create this folder that causes the error.
I will push my fix now.
